src/backend/joanie/edx_imports/tasks/certificates.py

```diff
@@ -138,10 +138,13 @@ def import_certificates_batch(start, stop, dry_run=False):
                 ],
             }
 
+        certificate_name = (
+            "degree" if edx_certificate.mode == "verified" else "certificate"
```

Comment:
There are enums value for existing certificate type (`joanie.core.enums`), we could use it here, WDYT ?

```suggestion
            DEGREE if edx_certificate.mode == "verified" else CERTIFICATE
```